### PR TITLE
Update docs with common templates and ballerina integrator download link

### LINKED ISSUE #483: Update docs with prerequisites, pull command and ballerina integrator link
**Description:**
- Add Prerequisites to docs. 
- Add pull commands to docs.
- Add ballerina integrator download link to docs.

### PR DESCRIPTION
**Pull request by dinuish94**
# Purpose
> - Add ballerina integrator download link.
> - Add prerequisites and ballerina pull to docs.
> - Resolves #483

### CODE REVIEW

**Comment by pramodya1994** [comment]
Did you build the site and preview it for this PR?

File: docs/content/src/learn/tutorials/integration-patterns-and-soa/eip-message-transformation/README.md [hunk 1]
```diff
@@ -46,10 +46,8 @@ The ultimate goal of the claim check EIP in our scenario is to validate the stud
 The sample scenario used to demonstrate the above three EI Patterns. Student sends a request with content student ID, student name, Student's city and gender. Then ‘content filter’ filters the student name, student's city and gender from the request and added them to a students details database. The filtered request contains only the student ID. Then filtered request goes to the 'Student ID validator'. It validates the incoming request and sends the validated request. Then validated request goes to the 'content enricher'. Content enricher uses used two databases as student's results details and student's personal details.
 Using student's ID, it maps the details which belong to a particular student from the tables and added particular data to the request. While adding the enriching data, it processes the data using JSON to JSON transformation. The output request of the scenario contains student ID, student name, Student's city, gender and student's results details.
 
-
-
 ## Prerequisites
- 
+
```
Comment (samgnaniah) [inline docs/content/src/learn/tutorials/integration-patterns-and-soa/eip-message-transformation/README.md]: Did we add the INCLUDE MD code in this topic?

File: docs/content/src/tutorial-prerequisites.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ## Prerequisites
  
-* Ballerina Integrator
+* [Ballerina Integrator](https://wso2.com/integration/?utm_source=facebook&utm_medium=organicpost&utm_campaign=landingpage_ei7_oct19&fbclid=IwAR3R_7pYwF4g5KIlxoitpav_nadIbCgkZwMLpaWqLfY7ZGRZZRHuSqbT6oQ#)
```
Comment (samgnaniah) [inline docs/content/src/tutorial-prerequisites.md]: Shall we use https://wso2.com/integration/ballerina-integrator/ ?

File: docs/content/src/learn/tutorials/integration-patterns-and-soa/eip-message-transformation/README.md (hunk 1 above)
Comment (dinuish94) [reply docs/content/src/learn/tutorials/integration-patterns-and-soa/eip-message-transformation/README.md]: This tutorial is not yet migrated to Ballerina 1.0.0

Comment (dinuish94) [reply docs/content/src/learn/tutorials/integration-patterns-and-soa/eip-message-transformation/README.md]: Hence the doc generator can't resolve the modules

**Pull request merged**